dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/subcommands/impl/All.java

```diff
@@ -34,11 +35,21 @@ public class All implements ExecutableWithNamespace {
     private DockerfileGitHubUtil dockerfileGitHubUtil;
 
     @Override
-    public void execute(final Namespace ns, final DockerfileGitHubUtil dockerfileGitHubUtil)
-            throws IOException, InterruptedException {
+    public void execute(final Namespace ns, final DockerfileGitHubUtil dockerfileGitHubUtil) {
         loadDockerfileGithubUtil(dockerfileGitHubUtil);
-        Set<Map.Entry<String, JsonElement>> imageToTagStore =
-                this.dockerfileGitHubUtil.getGitHubJsonStore(ns.get(Constants.STORE)).parseStoreToImagesMap(dockerfileGitHubUtil, ns.get(Constants.STORE));
+        String store = ns.get(Constants.STORE);
+        try {
+            ImageTagStore imageTagStore = ImageStoreUtil.initializeImageTagStore(this.dockerfileGitHubUtil, store);
+            List<ImageTagStoreContent> imageNamesWithTag = imageTagStore.getStoreContent(dockerfileGitHubUtil, store);
+            Integer numberOfImagesToProcess = imageNamesWithTag.size();
+            List<ProcessingErrors> imagesThatCouldNotBeProcessed = processImagesWithTag(ns, imageNamesWithTag);
+            printSummary(imagesThatCouldNotBeProcessed, numberOfImagesToProcess);
+        } catch (Exception e) {
+            log.error("Encountered issues while initializing the image tag store or getting its contents. Exception: ", e);
```

Comment:
Isn't this fatal? Why not throw? Seems like we should exit with non-zero exit code if this happens